modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -153,6 +153,16 @@ These adjustments are only applied to cleartext, edge-terminated, and re-encrypt
 
 For request headers, these adjustments are applied only for routes that have the `haproxy.router.openshift.io/h1-adjust-case=true` annotation. For response headers, these adjustments are applied to all HTTP responses. If this field is empty, no request headers are adjusted.
 
+|`tuningOptions`
+|`tuningOptions` specifies options for tuning the performance of Ingress Controller pods.
```

Comment:
Maybe remove "Setting this field is optional" from all the following descs and move it here once--"The following fields are optional."  (Sigh, the field name includes "options".)

Reply:
I can remove the optional line from each--there's a note following the table anyway.